ui/app/components/raft-storage-overview.js

```diff
@@ -16,12 +18,32 @@ export default Component.extend({
     if ('serviceWorker' in navigator) {
       // this checks to see if there's an active service worker - if it failed to register
       // for any reason, then this would be null
-      let worker = await navigator.serviceWorker.getRegistration(config.serviceWorkerScope);
+      const worker = await navigator.serviceWorker.getRegistration(config.serviceWorkerScope);
       if (worker) {
+        navigator.serviceWorker.addEventListener('message', this.serviceWorkerGetToken.bind(this));
+
         this.set('useServiceWorker', true);
       }
     }
   },
+  willDestroy() {
+    if (this.useServiceWorker) {
+      navigator.serviceWorker.removeEventListener('message', this.serviceWorkerGetToken);
+    }
+    this._super(...arguments);
+  },
+
+  serviceWorkerGetToken(event) {
+    const { action } = event.data;
+    const [port] = event.ports;
+
+    if (action === 'getToken') {
+      port.postMessage({ token: this.auth.currentToken });
```

Comment:
Do we still want to account for when/if the token isn't retrieved?

Reply:
It doesn't look like the original implementation was doing anything other than logging an error to the console. If there is no token the request would result in a 403. I'm not sure in practice if this would occur now since the user would be redirected to the auth route before the component is instantiated.